code/modules/surgery/organs/vocal_cords.dm

```diff
@@ -1082,15 +1082,15 @@
 			switch(E.phase)
 				if(1 to 2)
 					E.phase = -1
-					to_chat(C, "<span class='big warning'>You have no recollection of being enthralled by [E.master]!</b></span>")
+					to_chat(C, "<span class='big warning'>You suddenly forget [E.master]'s modulus operandi and their political affiliations!</b></span>")
```

Comment:
This isn't really too clear from the enthrallee's point of view.  The former flavortext was a lot more clear of exactly what is supposed to happen.

Reply:
Reverted